pkg/template/cloudconfig_test.go

```diff
@@ -55,8 +58,17 @@ func TestCloudConfig(t *testing.T) {
 		if err != nil {
 			t.Errorf("failed to render ignition files, %v:", err)
 		}
+		tc.params.Extension = nopExtension{}
 		tc.params.Files = files
-		tc.params.Images = BuildImages("docker.io", Versions{})
+		tc.params.Versions = Versions{
+			Calico:                       "3.14.1",
+			CRITools:                     "1.17.0",
```

Comment:
Why default a different version here than kubernetes if we use the same version constraint to verify?

Reply:
CRI-tools is kind of weird in that they only release one per Kubernetes minor, so 1.16.0, 1.17.0, etc. See https://github.com/kubernetes-sigs/cri-tools/releases I wanted a real version here in case we want to add more validation in code in the future. It feels arbitrary either way.

Reply:
Actually there is a cri-tools@1.16.1 so even that rule doesn't hold. I'll just make it match the Kubernetes version for now--maybe we can add a check that the image exists in the docker registry in the future.